objects.js

```diff
@@ -2640,87 +2640,135 @@ SpriteMorph.prototype.reporterize = function (expressionString) {
     // highly experimental Christmas Easter Egg 2016 :-)
     var ast;
 
-    function parseInfix(expression, operator, already) {
-        // very basic binary infix parser for arithmetic expressions
```

Comment:
I don't know if it matters much, but probably around createAST since that's doing essentially the same work.